quickstart.md

```diff
@@ -0,0 +1,174 @@
+
+```JavaScript
+myProfile = await idx.set('basicProfile', {
+  name: '<YOUR NAME> ',
+  emoji: '✌🏻', // most emoji's will work, if you get an error try a different one. Some are registered as 2 characters not one.
+  description: "<YOUR DESCRIPTION>"
+})
+```
+
+Updating a record is done using `idx.merge()`. The major difference between set & merge is that merge will only update the parameters given, set will set all parameters (even those not included).
+
+```JavaScript
+await idx.merge('basicProfile', {
+  description: '<NEW DESCRIPTION>'
+})
+```
+
+This can be checked quickly with `myProfile = await idx.get('basicProfile')`. You should see your changes reflected.
+
+As mentioned above IDX was created to access records that aren't only yours. The playground has been preloaded with `exampleDID` & `exampleDID2` variables for you to use.
+
+```JavaScript
+await idx.get('basicProfile', exampleDID)
+```
+
+You should now see a basicProfile in your terminal, all you needed is a DID and you can see my record. If you've been following along you might remember the `controller` parameter we set for our `ceramicBlogPost`, similar to how you are the only person who can edit that post or update it in anyway you will be unable to update my record. This is because you aren't the controller, but what if we were to try? 
+
+```JavaScript
+await idx.set(
+  'basicProfile', 
+  { emoji: '💻' }, 
+  exampleDID
+)
+```
+
+Now to confirm that nothing got changed let's run an `idx.get()`.
+
+```JavaScript
+await idx.get('basicProfile', exampleDid)
+```
+
+You'll notice that our change wasn't applied at all. Which is great because it shouldn't have applied. 
+
+## Getting our Blog Post ready for IDX
+Now with what we've learned about IDX, let's get our blog posts ready for use with IDX. This is a fairly simple process that we've helped make as straight forward as possible. 
+
+For starters we will need to create a [definition](https://developers.ceramic.network/tools/idx/overview/#definitions){:target="_blank"} to tell IDX what we're developing.
+
+```JavaScript
+const blogDefinition = await createDefinition(ceramic, {
+  name: 'Blog Post',
+  description: 'A single blog post',
+  schema: schema.commitId.toUrl() // Note this is a URL not a string. Ceramic urls are prefaced with ceramic://
+})
+```
+
+Now we're ready to use this definition to create a record! We just need to use our new commit id we created above to create our records that will be automatically added to your IDX record. Creating a Definition will also enforce the structure of your schema, ensuring that the data requested will be provided.
+
+```JavaScript
+const newBlogPost = await idx.set(blogDefinition.commitId.toString(), {
```

Comment:
I think this example could be confusing: usually blogs can have more than one post, but here we can only associate one post to the definition. It might not be obvious to people getting started that this will replace any previous post they had.
Maybe something else than a blog/post could be used as example?

Reply:
Good point, I've changed it very slightly to be books instead, I think that's a touch more tangible. That being said I'm open to any other ideas.